ci/docker/android-emulator/Dockerfile

```diff
@@ -1,20 +1,21 @@
-FROM nvidia/opengl:1.0-glvnd-runtime-ubuntu18.04
+FROM ubuntu:20.04
```

Comment:
20.04.2 ?

Reply:
it's basic image, that includes every latest update; but i agree, it's better to specify

Reply:
there are no such image Oo 
https://hub.docker.com/_/ubuntu?tab=tags&page=1&ordering=last_updated&name=20.04

Reply:
mkay